internal/autotag/gallery_test.go

```diff
@@ -46,12 +46,24 @@ func TestGalleryPerformers(t *testing.T) {
 		mockPerformerReader.On("QueryForAutoTag", testCtx, mock.Anything).Return([]*models.Performer{&performer, &reversedPerformer}, nil).Once()
 
 		if test.Matches {
-			mockGalleryReader.On("UpdatePartial", testCtx, galleryID, models.GalleryPartial{
-				PerformerIDs: &models.UpdateIDs{
-					IDs:  []int{performerID},
-					Mode: models.RelationshipUpdateModeAdd,
-				},
-			}).Return(nil, nil).Once()
+			matchPartial := mock.MatchedBy(func(got models.GalleryPartial) bool {
+				expected := models.GalleryPartial{
+					PerformerIDs: &models.UpdateIDs{
+						IDs:  []int{performerID},
+						Mode: models.RelationshipUpdateModeAdd,
+					},
+				}
+
+				// updated at should be set and not null
+				if !got.UpdatedAt.Set || got.UpdatedAt.Null {
+					return false
+				}
+				// else ignore the exact value
+				got.UpdatedAt = models.OptionalTime{}
+
+				return assert.Equal(got, expected)
```

Comment:
I think this can be reused in a utility function - there's a bit of repeated code in these.